### DISCUSSION TITLE
Welcome to did-btc-spec Discussions!

### DISCUSSION THREAD

**Discussion by sangaman** [MicroStrategy/did-btc-spec#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by monjikura** [comment]
monjikura

**Comment by amzinglyrwarded** [comment]
I would like to look into integrating this into my current android app, as a way to sign into the platform. Where do I begin?

**Reply by LeoBeb** [reply]
As of July 22nd, we have made available the source code of our SDK along with its documentation. You can find them under the following links:
Source code: https://github.com/MicroStrategy/did-btc-spec/tree/main/sdk
Documentation: https://microstrategy.github.io/did-btc-spec/sdk/docs/

**Comment by cameronwall** [comment]
Hi @sangaman , well done on Orange (BTC:DID). We have taken it one further after Michael's AusCryptoCon appearance in Australia.

We are blending Web2 <> Web3 technologies to safeguard digital assets ranging from simple phone PINs and login credentials to cryptocurrency private keys. Would love your feedback or to get involved. www.blockscribe.io

**Comment by cameronwall** [comment]
Hi @sangaman , well done on Orange (BTC:DID). We have taken it one step further after Michael's AusCryptoCon appearance in Australia last year.

We are blending Web2 <> Web3 technologies to safeguard digital assets ranging from simple phone PINs and login credentials to cryptocurrency private keys. Would love your feedback or to get involved? www.blockscribe.io
